styles.css

```diff
@@ -116,6 +110,7 @@ img {
   display: none;
 }
 
+/* media sizing */
```

Comment:
call this section something like "break points" or "responsive break points" and move it to the VERY END.

Reply:
Ammended this, the sections have been moved to fit the overall format.